themes/default/content/docs/intro/languages/yaml.md

```diff
@@ -80,6 +80,11 @@ outputs:
 The example is a fully valid and self-contained Pulumi project. You only need one file to create resources in Pulumi YAML.
 {{% /notes %}}
 
+Further examples are given in the [Pulumi YAML GitHub repository][]. The specification for Pulumi
+YAML documents is in the [Pulumi YAML reference]({{< relref "/docs/reference/yaml" >}}).
+
+[Pulumi YAML GitHub repository]: https://github.com/pulumi/pulumi-yaml/tree/main/examples
```

Comment:
This is totally valid of course, but I might suggest putting this URL right into the link up on 83, for a few reasons:

* It's clearer to someone who may see `[][]` and think it's invalid or not understand why/how it works
* It's used only once (whereas multiple references might make a stronger case for it)
* On pages containing language choosers, where multiple blocks of rendered Markdown may contain the same link name but use different URLs (it's happened), it can be hard to debug why a particular link on the page is pointing to the wrong URL

If this all sounds sane to you as well, I'm happy to propose an addition to the Markdown linter for it.

Reply:
I find this tidier and easier to read than inlining longish URLs; and the `[][]` syntax is regular markdown, so I wouldn't expect it to divert anyone for very long (and only once or twice :-). However: sticking to the house style takes priority over my individual taste, so I'm happy to change it.

I don't think it's worth adding to the linter though, since there are acceptable reasons to use this style (e.g., "multiple references might make a stronger case for it") and having to add/read past opt-outs would be more trouble than the thing you're trying to avoid.